habitat/plan.sh

```diff
@@ -43,7 +43,9 @@ do_before() {
 
 do_download() {
   build_line "Building gem from source. (${SRC_PATH}/${pkg_name}.gemspec)"
-  gem build "${SRC_PATH}/${pkg_name}.gemspec"
+  ( cd "${SRC_PATH}" || exit_with "unable to enter hab-src directory" 1
+    gem build "${SRC_PATH}/${pkg_name}.gemspec"
```

Comment:
If we `cd` into the `SRC_PATH`, I don't think we need to do the whole path here anymore.
```suggestion
    gem build "${pkg_name}.gemspec"
```